File: perl/tests/600-index-users.pl
my $admin          = &AdminHandle;
our $user_list_600 = $admin->index_users();

if ($DEBUG) {
  my $top_entries = &ShowTopEntries($user_list_600,4);
  print "LIST (first users)=\n${top_entries}\n";
}

die "Value returned not an array.\n" unless defined($user_list_600) && ref($user_list_600) eq "ARRAY";

1;

File: ruby/tests/600-index-users.rb
admin          = admin_handle
@user_list_600 = admin.index_users()

puts "LIST (first users)= \n#{show_top_entries(@user_list_600,5)}" if @debug

raise "Value returned by index_users() is not an array" unless @user_list_600.present? && @user_list_600.is_a?(Array)

true
